README.md

```diff
@@ -1,2 +1,3 @@
 # react-repo-example
-Our guide repository structure for react apps, that should be used as a template. 
+
+- Our guide repository structure for react apps, that should be used as a template.
```

Comment:
What about a longer description here about some technical feature like the `ci.yml`?